cirq-google/cirq_google/engine/engine_client_test.py

```diff
@@ -851,6 +847,59 @@ def test_run_job_over_stream(
     stream_manager.submit.assert_called_with(*expected_submit_args)
 
 
+@pytest.mark.parametrize(
+    'run_job_kwargs, expected_submit_args',
+    [
```

Comment:
Why is this test parameterized? It looks like there's only one list element (?).

https://docs.pytest.org/en/stable/example/parametrize.html